web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -235,7 +235,7 @@ export class AccountService {
       this.i18nService.getTranslation('youve_got_burst') :
       this.i18nService.getTranslation('you_sent_burst'),
       {
-        body: `${convertNQTStringToNumber(transaction.amountNQT)} BURST`,
+        body: `${convertNQTStringToNumber(transaction.amountNQT + transaction.feeNQT)} BURST`,
```

Comment:
:trumpet:

Reply:
convertNQTStringToNumber(transaction.amountNQT) + transaction.feeNQT ?

I sent 10 burst + 0.00735 fee. Notification said I sent: 1000000000.00735

Reply:
hm.... should not happen with this line.... testing it now

Reply:
can confirm this...need to investigate

Reply:
Ahhhhh... @blankey1337 we're concatenating strings here! `'10000000' + '73500'`

Reply:
Sorry! I will wear the cone of shame for the rest of today.

Reply:
:tada: